firedantic/tests/tests_async/test_model.py

```diff
@@ -95,6 +95,52 @@ async def test_find(configure_db, create_company, create_product):
         await Product.find({"product_id": {"<>": "a"}})
 
 
+@pytest.mark.asyncio
+async def test_find_not_in(configure_db, create_company):
+    ids = ["1234555-1", "1234567-8", "2131232-4", "4124432-4"]
+    companies = []
```

Comment:
seems like `companies` variable is not used anywhere. you can remove it and do just 
```
    for company_id in ids:
        await create_company(company_id=company_id)
```
I noticed the same in `test_find` btw

Reply:
Thanks, fixed now in both places.